### DISCUSSION TITLE
...Has been blocked by CORS policy...

### DISCUSSION THREAD

**Discussion by tarik-gadoumi** [codesandbox/sandpack#658] [Q&A]
Hey all, i'm trying to integrate sandpack code playground on my website. it work's Great !.

i don't understand why  i'm getting the following  error message on the DevTool 

**"Access to fetch at 'https://data.jsdelivr.com/v1/package/npm/@babel/runtime' from origin 'https://1-3-2-sandpack.codesandbox.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled."** 

Does anyone have any idea where this error comes from?

Thank you !

**Comment by danilowoz** [comment]
Hey @tarik-gadoumi, I'm sorry you're facing this issue.
Could you provide a sandbox where we can reproduce and debug this error?

Thanks

**Comment by rfenderdtn** [comment]
Any suggested work around or fix on this issue? I run into the same issue with private registries from jfrog that are fully authenticated correctly
